specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/restorePoint.json

```diff
@@ -775,12 +775,67 @@
           "description": "Gets the managed disk details"
         },
         "diskRestorePoint": {
-          "$ref": "./computeRPCommon.json#/definitions/ApiEntityReference",
+          "$ref": "#/definitions/DiskRestorePointProperties",
           "description": "Gets the disk restore point Id."
```

Comment:
We can change the description as it is not just disk restore point Id

Reply:
Changed